scripts/update-component-sha.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+if [ $# -ne 2 ] ; then
+    echo "Need <OLD> and <NEW> as arguments" >&2
+    exit 1
+fi
+old=$1
+new=$2
+
+git grep -l "$old" | xargs sed -i -e "s,$old,$new,g"
```

Comment:
Yes, bit out of place (other than the fact I did a lot of bulk updates in this PR) but figured it didn't deserve its own PR and I've been meaning to submit it for ages. https://github.com/linuxkit/linuxkit/pull/2159#issuecomment-313427313